app/mailers/thing_mailer.rb

```diff
@@ -1,9 +1,15 @@
 class ThingMailer < ActionMailer::Base
   default from: 'adoptadrain@codeforsanfrancisco.org'
 
+  def adopted_confirmation(thing)
+    @thing = thing
+    @user = thing.user
+    mail(to: @user.email, subject: ['Thank you for Adopting a Drain in San Francico'])
```

Comment:
I think we can separate this out into a separate issue and deal with translations as a separate stream of activity. The bones are there for technical implementation, I think the heavier lifting will be content generation.